### PR TITLE
Settings page for admin

### PR DESCRIPTION
**Pull request by AlexanderBabak**

### CODE REVIEW

File: src/features/common/hooks/useGetUser/useGetUser.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
 import React, { createContext, useContext, useReducer } from "react";
 import { useQuery } from "@apollo/react-hooks";
 import { jwtDecode } from "jwt-decode";
-import { GetUserByIdQuery, LoginUserMutation, User } from "../../../../__generated__/graphql";
+import { FeatureSwitch, GetUserByIdQuery, LoginUserMutation, User } from "../../../../__generated__/graphql";
```
Comment (GirbuNicolae) [inline src/features/common/hooks/useGetUser/useGetUser.tsx]: use absolute paths when possible

File: src/features/common/libs/utils/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { createSearchParams } from "react-router-dom";
+import { searchParamsToObject, withoutFalsyValues } from "./utils";
+
+describe("utils", () => {
+  beforeAll(() => jest.useFakeTimers());
+  afterAll(() => jest.useRealTimers());
+
+  it("searchParamsToObject() should work as expected", async () => {
```
Comment (GirbuNicolae) [inline src/features/common/libs/utils/utils.test.ts]: Make tests more descriptive: e.g: "searchParamsToObject should convert search params string array into object"

File: src/features/common/ui/molecules/MessageCard/MessageCard.test.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { render } from "@testing-library/react";
+import placeholder from "features/common/assets/images/image-placeholder.svg";
+import MessageCard from "./MessageCard";
+
+describe("MessageCard", () => {
+  it("should work as expected", async () => {
```
Comment (GirbuNicolae) [inline src/features/common/ui/molecules/MessageCard/MessageCard.test.tsx]: avoid naming "should work as expected", instead describe the expected behaviour

**Review by GirbuNicolae — APPROVED** [review]

**Comment by AlexanderBabak** [comment]
Hello @AlexanderBabak how are you? Test

**Pull request merged**